docs/deploying-airbyte/on-oci-vm.md

```diff
@@ -76,66 +40,61 @@ sudo service docker start
 sudo usermod -a -G docker $USER
 ```
 
+### 2. Install Docker Compose
 
-### Install Docker Compose
+In the terminal connected to your OCI Instance for Airbyte, run the following commands:
 
-```text
+```bash
 sudo wget https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m) -O /usr/local/bin/docker-compose
 
 sudo chmod +x /usr/local/bin/docker-compose
 
 sudo /usr/local/bin/docker-compose --version
 ```
 
+## Install and start Airbyte
 
-### Install Airbyte
-
-```text
-mkdir airbyte && cd airbyte
+Download the Airbyte repository and deploy it on the VM:
 
-wget https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml}
+1. Run the following commands to clone the Airbyte repo:
 
-which docker-compose
+	```bash
+	mkdir airbyte && cd airbyte
 
-sudo /usr/local/bin/docker-compose up -d
-```
+	wget https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml}
+	```
 
-### Airbyte URL Access Method 1 : Local Port Forward to Airbyte VM using Bastion host
+2. Run the following commands to get Airbyte running on your OCI VM instance using Docker compose:
 
-```text
-ssh opc@bastion-host-public-ip -i <private-key-file.key> -L 8000:oci-private-instance-ip:8000
-```
+    ```bash
 
-### Access Airbyte
+    which docker-compose
 
-Open URL in Browser : [http://localhost:8000/](http://localhost:8000/)
+    sudo /usr/local/bin/docker-compose up -d
 
-![](../.gitbook/assets/OCIScreen4.png)
+    ``` 
 
-### Airbyte URL Access Method 2 : Local Port Forward to Airbyte VM using OCI Bastion Service
+3. Open up a Browser and visit port 8000 - [http://localhost:8000/](http://localhost:8000/)
```

Comment:
Change "Open up a Browser and visit port 8000 - [http://localhost:8000/](http://localhost:8000/)" to "On your browser, visit [port 8000](http://localhost:8000/)"